scanpointgenerator/rois/polygonal_roi.py

```diff
@@ -35,7 +35,7 @@ def mask_points(self, points):
         x = points[0]
         y = points[1]
         v1 = self.points[-1]
-        mask = np.full(len(x), False, dtype=bool)
+        mask = np.full(len(x), False, dtype=np.int8)
```

Comment:
sizeof(np.bool) == sizeof(np.int8)
No more than it used to. The parameters passed in take up much more space since they're arrays of doubles.